src/main/java/net/discordjug/javabot/util/WebhookUtil.java

```diff
@@ -118,11 +128,11 @@ private static CompletableFuture<ReadonlyMessage> mirrorMessageToWebhook(@NotNul
 		message.addEmbeds(embeds.stream()
 				.map(e -> WebhookEmbedBuilder.fromJDA(e).build())
 				.toList());
-		List<Attachment> attachments = originalMessage.getAttachments();
+		List<Attachment> newAttachments = attachments != null ? attachments : originalMessage.getAttachments();
```

Comment:
Please change this to use the positive version of the condition as this is easier readable:
```java
List<Attachment> newAttachments = attachments == null ? originalMessage.getAttachments() : attachments;
```
Or even better:
```java
List<Attachment> newAttachments = Objects.requireNonNullOrElse(originalMessage.getAttachments());
```

(Please use the second version if possible - I just included the first version as a general statement that positive statements (reducing unnecessary negation) are more easily readable than negative statements)